srcpkgs/limine/template

```diff
@@ -10,7 +10,7 @@ maintainer="Mintsuki <mintsuki@protonmail.com>"
 license="BSD-2-Clause"
 homepage="https://limine-bootloader.org/"
```

Comment:
```suggestion
homepage="https://limine-bootloader.org/"
changelog="https://raw.githubusercontent.com/limine-bootloader/limine/trunk/ChangeLog"
```

Reply:
Addressed, thanks.

Reply:
sorry for being nitpicky, but I think using a general development branch for the changelog (instead of the `v${version}`) is better here because it lets onlookers get a better overview, unless `trunk` is not a good candidate for some reason?

Reply:
No problem. I just thought that passing the ChangeLog associated with the version of the recipe would make more sense, unless this is something more like the homepage where expectations of the content being associated with the given version is nil.

Reply:
yes, I think it's the latter, I believe we accept versioned changelog urls mostly if there's no better source of changelog otherwise

Reply:
There, sorry about the confusion.